metatron-app/metatron-discovery
# #3816 user custom dim color

### Description

You can specify color for dimension values for Bar, Scatter, Pie, and WordCloud.

**Related Issue** : #3816 


### How Has This Been Tested?

1. Create Bar, Scatter, Pie, and WordCloud.
2. Change the color settings for the chart to the color settings for dimension values.
3. Check if you can specify a separate color for each dimension value.

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
